cpp/tests/utilities_tests/column_utilities_tests.cpp

```diff
@@ -208,6 +214,34 @@ TYPED_TEST(ColumnUtilitiesTestNumeric, PrintColumnWithInvalids)
   EXPECT_EQ(cudf::test::to_string(cudf_col, delimiter), tmp.str());
 }
 
+TYPED_TEST(ColumnUtilitiesTestFloatingPoint, PrintColumnNumeric)
```

Comment:
Is there a separate code path when there are no invalids in the column? To me it seems like the test without invalid values is redundant at can be removed if the PrintColumnWithInvalids tests also includes the two values removed from the column (2 and 4e-08).

Reply:
The code paths do diverge slightly. The nullable case branches _outside_ of the loop so it doesn't have to check nullness _inside_ the loop. What if we keep the existing test cases and remove the extras if/when we come back to refactor?